frontend/src/app/components/wp-fast-table/builders/drag-and-drop/drag-drop-handle-builder.ts

```diff
@@ -4,6 +4,9 @@ import {Injector} from "@angular/core";
 import {TableDragActionsRegistryService} from "core-components/wp-table/drag-and-drop/actions/table-drag-actions-registry.service";
 import {TableDragActionService} from "core-components/wp-table/drag-and-drop/actions/table-drag-action.service";
 
+/** Debug the render position */
+const RENDER_DRAG_AND_DROP_POSITION = false;
```

Comment:
To get an insight into what the relative position is for a work package (useful for debugging and reviewing), set this to `true`.